event-statistics/src/main/kubernetes/minikube.yml

```diff
@@ -94,6 +94,11 @@ spec:
       name: apicurio
   template:
     metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
```

Comment:
Shouldn't is be /q/metrics?

Reply:
I can't find any documentation about it on apicurio, but if I `docker run -it -p 8080:8080 apicurio/apicurio-registry-mem:2.1.5.Final`, doing `http :8080/metrics` gets me the metrics and `http :8080/q/metrics` gets me a `404`